client/styles/tag.module.css

```diff
@@ -0,0 +1,35 @@
+.tag_style {
```

Comment:
Note: change the name of this file to `Tag.module.css` with a capital `T` to match the naming of the other files

Reply:
Also note, if you change the className of the `<a></a>` element, also change it in this CSS file